stream_alert_cli/helpers.py

```diff
@@ -203,3 +205,68 @@ def put_mock_s3_object(bucket, key, data, region):
         Key=key,
         ServerSideEncryption='AES256'
     )
+
+
+def mock_me(context):
+    """Decorator function for wrapping framework in mock calls
+    for running local tests, and omitting mocks if testing live
+
+    Args:
+        context (namedtuple): A constructed aws context object
+    """
+    def wrap(func):
+        """Wrap the returned function with or without mocks"""
+        if context.mocked:
+            @mock_cloudwatch
+            @mock_lambda
+            @mock_s3
```

Comment:
Just beware of https://github.com/spulec/moto/issues/1026; the mock may not be restored correctly every time